request_handler.gemspec

```diff
@@ -28,8 +28,9 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'confstruct', '~> 1.0.2'
   spec.add_dependency 'multi_json', '~> 1.12'
+  spec.add_dependency 'activesupport', '> 4.0'
```

Comment:
we need this for what?

Reply:
we had deep_symbolize_keys implemented ourselfes. i figured it would be easier to use it from activesupport

Reply:
we implemented our-self to not need this runtime dependency

Reply:
last time we spoke you said implementing stuff like this ourselfes makes no sense since we can use activesupport and its used in almost all projects anyways... And I kinda agree with that version more that with the implement it ourselves version. Do you want me to revert that change?

Reply:
In our internal apps we have activesupport everywhere so for internal gems we might as well use it. For this the need was so little to just add the method directly instead of adding the dependency also because that way we don't push this dependency onto users outside ourselves.

but we can keep it in as well

Reply:
ok i'll keep it in then